fuel-vm/src/interpreter/diff/storage.rs

```diff
@@ -0,0 +1,474 @@
+use std::collections::HashMap;
+use std::fmt::Debug;
+
+use fuel_types::AssetId;
+use fuel_types::Bytes32;
+use fuel_types::ContractId;
+
+use crate::storage::InterpreterStorage;
+
+use super::ExecutableTransaction;
+use super::Interpreter;
+use super::*;
+
+#[derive(Debug)]
+/// The set of state changes that are recorded.
+pub(super) enum StorageDelta {
+    State(MappableDelta<(ContractId, Bytes32), Bytes32>),
+    Assets(MappableDelta<(ContractId, AssetId), u64>),
+    Info(MappableDelta<ContractId, (fuel_types::Salt, Bytes32)>),
```

Comment:
[nit] might be more readable as `ContractInfo`

```suggestion
    ContractInfo(MappableDelta<ContractId, (fuel_types::Salt, Bytes32)>),
```

Reply:
If this change is made I would also `ContractState` and `ContractState` for consistency

Reply:
Yeh and then they all end up with contract infront